Source/carthage/Archive.swift

```diff
@@ -103,6 +103,8 @@ public struct ArchiveCommand: CommandType {
 						.map { ($0 as NSString).lastPathComponent }
 						.filter { $0.hasSuffix(".framework") }
 
+					let relativePaths = paths.map { $0.stringByReplacingOccurrencesOfString(options.directoryPath, withString: ".") }
```

Comment:
Do you think this is a better solution?

![carthage-archive-issue](https://cloud.githubusercontent.com/assets/346012/17923218/5f9a3a50-69e3-11e6-9b3a-e1b30dfdcc37.png)